sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/QBiCPortletUI.groovy

```diff
@@ -0,0 +1,101 @@
+     * Location of the local configuration. This is useful when testing portlets loocally using {@code mvn jetty:run}.
+     */
+    public static final String DEVELOPER_PROPERTIES_FILE_PATH = "developer.properties";
+
+    static final String INFO_LABEL_CLASS = "portlet-footer";
+    static final String INFO_LABEL_ID = "qbic-portlet-info-label";
+    static final String DEFAULT_REPO = "http://github.com/qbicsoftware";
+    static final String DEFAULT_VERSION = "0.0.1-alpha";
+
+
+    private static final Logger LOG = LogManager.getLogger(QBiCPortletUI.class);
+
+    private final String portletVersion;
+    private final String portletRepoURL;
+
+    /**
+     * Default constructor.
+     */
+    public QBiCPortletUI() {
+        // load url/version from portlet.properties
+        LOG.info("Initializing QBiCPortletUI");
+        final Properties portletProperties = new Properties();
+        try (final InputStream propertiesFileStream =
+                QBiCPortletUI.class.getClassLoader().getResourceAsStream(PORTLET_PROPERTIES_FILE_PATH)) {
+            portletProperties.load(propertiesFileStream);
+            if (!portletProperties.containsKey("version")
+                    || !portletProperties.containsKey("repository.url")) {
+                LOG.error("Missing version and/or repository url properties in portlet.properties file. "
+                        + "The file should contain the 'version' and 'repository.url' properties. "
+                        + "Using default version and/or repository url.");
+            }
+        } catch (final Exception e) {
+            LOG.error("Error loading portlet.properties. Portlet will display bogus version/url values. "
+                    + "Check that portlet.properties is found under the /WEB-INF/classes folder (this file "
+                    + "is copied from src/main/resources/portlet.properties during the build).", e);
+        }
+
+        portletVersion =
+                StringUtils.trimToEmpty(portletProperties.getProperty("version", DEFAULT_VERSION));
+        portletRepoURL =
+                StringUtils.trimToEmpty(portletProperties.getProperty("repository.url", DEFAULT_REPO));
+    }
+
+    @Override
+    protected final void init(final VaadinRequest request) {
+        LOG.info("Version " + portletVersion);
+        final VerticalLayout layout = new VerticalLayout();
+        layout.setMargin(false);
+        // add the portlet
+        layout.addComponent(getPortletContent(request));
+
+        addPortletInfo(layout);
+        setContent(layout);
+    }
+
+    @Override
+    public final void setContent(final Component content) {
+        super.setContent(content);
+    }
+
+    // adds version and repository information to the passed layout
+    private void addPortletInfo(final Layout layout) {
+        final Label portletInfoLabel = new Label("version " + portletVersion + " (<a href=\""
+                + portletRepoURL + "\">" + portletRepoURL + "</a>)", ContentMode.HTML);
```

Comment:
![Bildschirmfoto 2021-06-10 um 14 25 17](https://user-images.githubusercontent.com/29627977/121524585-f8dfa700-c9e6-11eb-9f4d-04f087f6462d.png)
The link breaks since it includes the child module in the link. 
This can be fixed by specifying the parent url explictly in the child(sample-tracking-status-overview-app) pom.xml file: 
`<url>http://github.com/qbicsoftware/Sample-Tracking-Status-Overview</url>`

Reply:
thanks for pointing this out, will do this